apps/ledger-live-mobile/src/components/ServicePermissionDeniedView.tsx

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import styled from "styled-components/native";
+import { Text, Flex } from "@ledgerhq/native-ui";
+import { ArrowLeftMedium } from "@ledgerhq/native-ui/assets/icons";
+import { TouchableOpacity } from "react-native-gesture-handler";
+import { IconType } from "@ledgerhq/native-ui/components/Icon/type";
+import { useNavigation } from "@react-navigation/native";
+import Button from "./Button";
+
+type Props = {
+  title: string;
+  TitleIcon?: React.ReactNode;
```

Comment:
if it's a ReactNode, don't capitalize, only capitalize if it's a React Component (something that you can pass props to)